common/lib/xmodule/xmodule/course_module.py

```diff
@@ -920,6 +920,20 @@ class CourseFields(object):
         scope=Scope.settings, default=None
      )
 
+    custom_block_type_keys = List(
+        display_name=_("Custom Block Types"),
+        help=_("Specify what kind of blocks will be mark as completed."),
+        default=[],
+        scope=Scope.settings
+    )
+
+    required_activity_ids = List(
+        display_name=_("Required Activity Ids"),
+        help=_("Specify what Activities will be required for the course"),
```

Comment:
this is not activities that is required for the course, it is specific to the course completion report, please let the message reflect that to not confuse the users,